etc/make_release.py

```diff
@@ -90,6 +95,9 @@
               help='Instead of building the C driver, use the one installed at this path')
 @click.option('--dist-file',
               help='Don\'t build anything; use this C++ driver distribution tarball')
+@click.option('--skip-distcheck',
+              is_flag=True,
+              help="Only build the distribution tarball (do not build the driver or run tests)")
```

Comment:
When debugging issues during release, and repeatedly running `make_release.py` we found it easier to iterate by only producing the tarball, and not running the full `distcheck`, which takes several minutes to build the C++ driver and run tests.